pkg/diskboot/config.go

```diff
@@ -63,7 +64,7 @@ type Entry struct {
 
 // KexecLoad calls the appropriate kexec load routines based on the
 // type of Entry
-func (e *Entry) KexecLoad(mountPath, appendCmdline string, dryrun bool) error {
+func (e *Entry) KexecLoad(mountPath string, filterCmdline cmdline.Filter, dryrun bool) error {
```

Comment:
I don't know about this -- I think there should still be an appendCmdline.

Reply:
The filter includes it, I could add a new filter that does append only if you prefer ?

Reply:
ah alright.